pkgs/applications/virtualization/crosvm/default.nix

```diff
@@ -1,48 +1,35 @@
-{ stdenv, lib, rust, rustPlatform, fetchgit, fetchpatch
-, clang, pkg-config, protobuf, python3, wayland-scanner
+{ stdenv, lib, rustPlatform, fetchgit
```

Comment:
`stdenv` is unused now too.  I'll update to drop that when I get the chance.

Reply:
I think you fixed this, right?

Oh totally awesome, I now get the "resolve" button on other peoples' comments.  Hooray.